concent_api/verifier/utils.py

```diff
@@ -145,10 +145,14 @@ def delete_file(file_path: str) -> None:
 
 def generate_full_blender_output_file_name(scene_file: str, frame_number: int, output_format: str) -> str:
     base_blender_output_file_name = generate_base_blender_output_file_name(scene_file)
-    return f'{base_blender_output_file_name}{frame_number:>04}.{output_format}'
+    return f'{base_blender_output_file_name}{frame_number:>04}.{output_format.lower()}'
```

Comment:
Currently `.lower()` is also called in the only place where this function is used (`render_images_by_frames` function`), so please remove it from there - it is now called twice, and off course it is better when it is called inside `generate_full_blender_output_file_name`.

Reply:
Done